packages/cms/src/utils/varSwap.js

```diff
@@ -27,7 +30,7 @@ module.exports = (sourceString, formatterFunctions, variables) => {
         return formatter(value, libs, formatterFunctions);
       }
       catch (e) {
-        console.log("Formatter error: ", e.message);
+        if (verbose) console.error("Formatter error: ", e.message, formatter);
```

Comment:
- don't put this behind `verbose` (we will probably always want to show errors on prod servers)
- let's do more explicit text, something like `Error using formatter "${formatter.name}" with variable ${value}`
- and then `e.message` on the next line?

Reply:
Done